packages/frontpage/lib/data.ts

```diff
@@ -379,25 +405,27 @@ export async function deletePost(rkey: string) {
 }
 
 type CommentInput = {
-  content: string;
   subjectRkey: string;
+  subjectCid: string;
+  subjectCollection: string;
+  content: string;
 };
 
-export async function createComment({ subjectRkey, content }: CommentInput) {
+export async function createComment({
+  subjectRkey,
+  subjectCid,
+  subjectCollection,
```

Comment:
I changed the input to createComment to match createVote. This'll be needed for replies anyway and thought it was best to just have them consistent to avoid any confusion.